### PR TITLE
task 7 implemented

### PR DESCRIPTION
**Pull request by jegius**
- [x] 1 - authorization-service is added to the repo, has correct basicAuthorizer lambda and correct serverless.yaml file
- [x] 3 - import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda. Request to the importProductsFile lambda should work only with correct authorization_token being decoded and checked by basicAuthorizer lambda. Response should be in 403 HTTP status if access is denied for this user (invalid authorization_token) and in 401 HTTP status if Authorization header is not provided.

- [ ] 5 - update client application to send Authorization: Basic authorization_token header on import. Client should get authorization_token value from browser localStorage https://developer.mozilla.org/ru/docs/Web/API/Window/localStorage authorization_token = localStorage.getItem('authorization_token')

https://d20wes0bg8lspj.cloudfront.net/admin/products
password = TEST_PASSWORD

### CODE REVIEW

File: authorization-service/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+jegius=TEST_PASSWORD
```
Comment (SkorykV) [inline authorization-service/.env:1]: why did you add env file here?

**Review by SkorykV — APPROVED** [review]
Overall looks good

File: authorization-service/serverless.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        }
+    },
+    plugins: [
+        'serverless-webpack',
+        'serverless-dotenv-plugin'
+    ],
+    provider: {
+        name: 'aws',
+        runtime: 'nodejs12.x',
+        iamRoleStatements: [
```
Comment (DzmitryU) [inline authorization-service/serverless.ts:21]: I don't think that we need access to all this resources from authorization lambda
